docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
+        {
+            this.IsSplitViewPaneOpen = !this.IsSplitViewPaneOpen;
+        });
+    }
+}
+```
+Now, to make the button invoke the command you just need to bind it. (For more informations see [here](https://docs.avaloniaui.net/docs/0.10.x/data-binding/binding-to-commands))
+``` xml
+<Button Command="{Binding ChangeSplitViewPaneStatusCommand}">
+```
+By doing so the command will be invoked each time the button is clicked and will update the IsSplitViewPaneOpen's value accordingly.
+<br><br>
+**If you followed this guide step by step you should now have something like this:**
+<img className="screenshot-full" src={SecondExample} alt="Second example" />
 
+## Adding content to the button
+Finally, you now just need to add some content to the button that indicates whether you need to show or hide the SplitView's pane. In this guide we will be using the following characters:
+- '<' to indicate the pane will be shown on click.
+- '>' to indicate the pane will be hidden on click.
+
+### Implement a converter
+To do this step we will need the char to change as the Pane goes from shown to hidden and viceversa. To do so we need to create a converter function that will return the necessary char to the button when clicked.<br>
+In the `models` folder create a class named SplitViewIconConverter and make it inherit from the IMultiValueConverter interface.<br>
+:::info
+I will use a MultiValueConverter because we need it to be called whenever the IsSplitViewPaneOpen value is changed. (For more information on how IMultiValueConverter behaves see [here](https://docs.avaloniaui.net/docs/guides/data-binding/how-to-bind-multiple-properties))  
+:::
+<br>
+First, implement the interface as follows:
+``` C#
+public object? Convert(IList<object?> values, Type targetType, object? parameter, CultureInfo culture)
+{
+    throw new NotImplementedException();
+}
+```
+Second, define how the method will behave based on the value which will be passed to it in the IList object. We will assume that the first value of the list will be the value of `IsSplitViewPaneOpen`, and will treat it accordingly. When the value is true the method will return '>', otherwise it will return '<'.
+``` C#
+public object? Convert(IList<object?> values, Type targetType, object? parameter, CultureInfo culture)
+{
+    if(values.Any(x => x is null or UnsetValueType or not bool))
+        return BindingOperations.DoNothing;
+
+    bool value = (bool)values[0];
+
+    if (value)
+        return ">";
+
+    return "<";
+}
+```
+:::warning
+Since we need only one value in this code we will assume that every single value passed to this function should be a boolean. 
+:::
+### Bind the converter to the button content.
+Lastly, all that's left to do is bind the converter to the button's content. To do so import the namespace of the converter in the MainWindow.axaml file by adding the following line
+``` xml
+xmlns:convs="clr-namespace:how_to_show_and_hide_a_split_view_pane_with_mvvm.Models"
```

Comment:
This is the first mention of the namespace. Anybody following the guide up until this point will likely have a different namespace

Reply:
Do you think it's bette if I put something like `"clr-namespace:YourNamespace.Models"`?

Reply:
You can do that as long as you mention to replace it. Alternatively, you can also show your namespace in other places in the guide so if people start blindly copying and pasting it will also show up.